src/components/atoms/AddComponentButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Card, Text } from 'evergreen-ui';
+import { dashboardItemsVar } from '../../cache/cache';
+import { WEATHER_MET_API } from '../../queries/metApi';
+
+const AddComponentButton: React.FC = () => {
+    const addDashboardItem = () => {
+        dashboardItemsVar([...dashboardItemsVar(), { size: 3, withText: false, query: WEATHER_MET_API }]);
```

Comment:
Ref. discussion with Jonathan, do you need `withText`? Or can you infer it from the content of the data/components?

Reply:
changed the type to description? string